<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateEditorPanel</class>
 <widget class="QMainWindow" name="TemplateEditorPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="nameEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="templateEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionDelete"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../qrc/icons/icons.qrc">
     <normaloff>:/add_template.svg</normaloff>:/add_template.svg</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../../qrc/icons/icons.qrc">
     <normaloff>:/remove_template.svg</normaloff>:/remove_template.svg</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionOpen_Drawer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../qrc/icons/icons.qrc">
     <normaloff>:/box.svg</normaloff>:/box.svg</iconset>
   </property>
   <property name="text">
    <string>Open Drawer</string>
   </property>
  </action>
  <action name="actionFavor">
   <property name="text">
    <string>Favor</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../qrc/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
